cmd/controller/main.go

```diff
@@ -19,14 +19,19 @@ package main
 
 import (
 	"fmt"
-	"github.com/vmware/purser/pkg/controller/persistence/dgraph"
+
+	"github.com/vmware/purser/pkg/persistence/dgraph"
```

Comment:
DB Persistence module is used by controller module only, do we want to keep it outside of it?

Reply:
okay.

Reply:
Even appD code will be part of controller, so it will be better if we keep this code inside controller. What do you think?.